app/views/static_pages/home.html.erb

```diff
@@ -1,10 +1,27 @@
 <% provide :title, t(".home_title") %>
-<div class="center jumbotron">
-  <h1><%= t ".home_title"  %></h1>
-  <p>
-    <%= t".home_html", link: link_to(t(".home_title"), Settings.link.home) %>
-  </p>
-  <%= link_to t(".sign_up"), signup_path, class: "btn btn-lg btn-primary" %>
-</div>
-<%= link_to image_tag("rails_logo.png", alt: "Rails logo", width: "#{Settings.size.logo}px"),
-"https://rubyonrails.org/" %>
+<% if logged_in? %>
+  <div class="row">
+    <aside class="col-md-4">
+      <section class="user_info">
+        <%= render 'shared/user_info' %>
+      </section>
+      <section class="micropost_form">
+        <%= render 'shared/micropost_form' %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3>Micropost Feed</h3>
+      <%= render 'shared/feed' %>
+    </div>
+  </div>
```

Comment:
nếu người dùng đăng nhập thì sẽ in ra thông tin người dùng, form để tạo bài viết,  danh sách các bài viết trước đó